packages/styles/src/banner/_banner.scss

```diff
@@ -0,0 +1,32 @@
+.as-banner {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  width: 100%;
+  padding: 12px;
+  border-radius: 0;
+  background-color: var(--as-color-ui-04, #{$color-ui-04});
+  font: $font-body;
+
+  &__content {
+    flex: 1;
+    padding: 0 8px;
+  }
+
+  &__icon {
+    display: flex;
+    flex: 0 0 auto;
+  }
+
+  &--support-01 {
```

Comment:
Yep, my first thought was to use `--success`, `--warning` and `--error`, but since we used the color names (`--type-01`) in the typography modifiers I decided to keep it that way.

In the case of support colors it's easy, you have `success`, `warning`, and `error`, but what about the typography? I can't think of 4 names for them, maximum 2 😂

Reply:
we could greate semantic aliases

--support-02 == warning
--type-04 == gray